job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext.groovy

```diff
@@ -350,7 +358,6 @@ class PublisherContext implements Context {
      *                 </hudson.plugins.plot.PropertiesSeries>
      *                 <hudson.plugins.plot.XMLSeries>
      *                     <file>XML</file>
-     *                     <label></label>
```

Comment:
The label element is generated for all types of series. Lame, but we should emulate that to keep the diff small.